### ISSUE TITLE
Open AI Generator

### ISSUE THREAD

**Issue by SohamRoy-01** [accodes21/threejs-react#1]
The openai generator is not working

**Comment by accodes21** [comment]
Yeah, I checked the credits for the api have finished so we need to create new account for it perhaps

**Comment by SohamRoy-01** [comment]
**Even creating a new account also OpenAi not giving free credits**
